CESC-submission/paper.tex

```diff
@@ -128,37 +128,37 @@ \section{Distributed Resource Markets Design}
 
 
-\noindent{\bf Financial and Cryptographic Security Measures.} Usually, security threats are mitigated by using cryptographic means (e.g., encryption and digital signatures), or algorithmic approaches (e.g., ordering the actions in a way that enforces secure behavior). Monetary-incentivized systems introduce new types of attacks that cannot be addressed using conventional approaches. In particular, having financially-motivated attackers introduces new threat vectors that need to be mitigated by using financial means. These fall into three categories: first, detect-and-punish mechanisms, where any participant is required to lock a penalty deposit that is forfeited if he/she is caught cheating. Second, designing algorithms that, if performed maliciously, require larger amounts of resources than when performed honestly. And third, designing service pricing policies that make it more profitable on the long run to act honestly in every service request than cheating (even if cheating is not detected). Such techniques make cheating unprofitable so that rational parties will choose to follow the protocol.
+\noindent{\bf Financial and Cryptographic Security Measures.} Usually, security threats are mitigated by using cryptographic means (e.g., encryption and digital signatures), or algorithmic approaches (e.g., ordering the actions in a way that enforces secure behavior). Monetary-incentivized systems introduce new types of attacks that cannot be addressed using conventional approaches. In particular, having financially-motivated attackers introduces new threat vectors that need to be mitigated by using financial means. These fall into three categories: first, detect-and-punish mechanisms, where any participant is required to lock collateral that is forfeited if he/she is caught cheating. Second, designing algorithms that, if performed maliciously, require larger amounts of resources than when performed honestly. And third, designing service pricing and payments mechanisms that make it more profitable on the long run to act honestly in every service request than cheating or ignoring the request (even if cheating or ignoring are not detectable). Such techniques make cheating unprofitable so that the majority of rational parties will choose to follow the protocol.
```

Comment:
penalty deposit -> collateral 
add 
- payment mechanisms 
- ignoring the request